.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly" 
+      time: "04:00"
+    open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: "typescript"
```

Comment:
This dependency doesn't exist on this project 
You can include the ones that have in package.josn the symbols `^` and `~`